Docs/src/ogre-shadows.md

```diff
@@ -397,15 +400,136 @@ Non-PCF block. Note that after doing this, the uSTexWidth and
 uSTexHeight variables are likely to be optimized away and so you should
 uncomment these variables in the materials script as well.
 
-The following shows how to activate plane optimal shadow mapping given
-some pointer to a MovablePlane and a pointer to a light.
+## Debugging Shadows
+Since shadows are a difficult subject, so it is a good idea to have the Shadow Map 
+projected on a Mini-Screen where it is possible to see how the Depth Caster is performing.
+
+```cpp
+// Create vertex program definition
+auto vertexProgram = Ogre::HighLevelGpuProgramManager::getSingleton().createProgram(
```

Comment:
using a .material script here would be probably more readable.

Also, I dont think you need to write any shaders yourself - the RTSS should be able to display the texture just fine.

Reply:
Yes, but I want to leave some documentation on how to do these kind of stuff and also to do things without the RTSS as well. (The ideal is to use the RTSS but it is algo good to have the knowlede of the underlying complexity).

Reply:
true, but I dont think it is a good idea to teach about underlying complexity of the RTSS in a section called "Debugging Shadows".

Instead, started adding API examples to the respective script pages here:
https://github.com/OGRECave/ogre/pull/2386/files

Reply:
Great. Yes, that is what I was talking about.